core/fs.ts

```diff
@@ -118,8 +118,15 @@ export default class FS {
 
   #walkFs(dir: Entry) {
     const dirPath = posix.join(this.options.root, dir.path);
+    let dirEntries
 
-    for (const dirEntry of Deno.readDirSync(dirPath)) {
+    try {
+      dirEntries = Deno.readDirSync(dirPath);
+    } catch {
+      console.error(`Error: The system cannot find the path at ${dirPath}.`)
+      Deno.exit()
+    }
+    for (const dirEntry of dirEntries) {
```

Comment:
Out of curiosity, what's the purpose of this? In what scenario the `Deno.readDirSync` can fail?
Just want to understand the motivation of this.

Reply:
Oh, it's unrelated to this PR. It should be on another PR, but my git skills was not fluent enough to split it to another branch yet still existing in the code for me to test, so I put it there and then forgot to take it out when pushed. I'll revert that when I work on my computer later.

It's just to give a nicer error when the user specify a source folder but it doesn't exist.